### ISSUE TITLE
Class ordering issue

### ISSUE THREAD

**Issue by strygo** [bridgedotnet/Bridge#883]
This seems related to issue #798. I've been able to distill my problem down to the following:

```
namespace ClassLibrary4
{
    using System;

    public class Class1 : Class2
    {
    }

    public interface IInterface
    {
    }

    public class Class2 : IInterface
    {
    }
}
```

This emits:

```
Bridge.define('ClassLibrary4.Class1', {
    inherits: [ClassLibrary4.Class2]
});

Bridge.define('ClassLibrary4.IInterface');

Bridge.define('ClassLibrary4.Class2', {
    inherits: [ClassLibrary4.IInterface]
});
```

This is problematic because ClassLibrary4.Class2 is not yet defined when Bridge.define() is invoked for Class1. (In this sample, ClassLibrary4 isn't yet defined, but in a variation of this, the inherits array can contain an undefined item which in turn results in an exception being thrown in Bridge.define())

**UPD:** One more report http://forums.bridge.net/forum/bridge-net-pro/bugs/1657

**Comment by geoffreymcgill** [comment]
Hi @strygo, 

The `[Priority]` Attribute can help work around this condition. 

http://bridge.net/kb/attribute-reference/#Priority

We're hoping to handle this automatically in a future release, although at the moment the attribute needs to be applied. 

Hope this helps.

**Comment by strygo** [comment]
Yes it does. I was able to use that to workaround my issue. Thanks!

**Comment by vladsch** [comment]
Thanks for the report, the issue will be fixed in 1.11 release
